README.md

```diff
@@ -42,15 +42,15 @@ sudo apt install llvm clang bindgen pkg-config libssl-dev libsqlite3-dev
 To install for production use cases, run:
 
 ```sh
-cargo install --path node
+cargo install --path bin/node
 ```
 
 This will install the executable `miden-node` in your PATH, at `~/.cargo/bin/miden-node`.
 
 Otherwise, if only to try the node out for testing, run:
 
 ```sh
-cargo install --features testing --path node
+cargo install --features testing --path bin/node
 ```
 
 Currently, the only difference between the two is how long the `make-genesis` command will take to run (see next subsection).
```

Comment:
This would be the right command to use in case you would want to run the software using the cloned repo. 

If you would like to use the binary directly installed from crates.io then yes indeed `cargo install miden-node` would be the right command. 

I believe that both command could / should be documented for completeness ( having multiple ways to download / install the software ) 

Let me know what is your pov.

Reply:
I think for `cargo install` downloading from crates.io is probably the main use case - but I'm not against also having a section which describes how to deploy a node by cloning the repo. In general, we should probably update the section about deploying the node because we now have multiple ways (installing from crates.io, via docker, via `.deb` package, by cloning the repo) and we should describe in a more comprehensive way than we do now.